okhttp-clients/src/main/java/com/palantir/remoting3/okhttp/OkHttpClients.java

```diff
@@ -73,25 +73,28 @@
     /** Global {@link TaggedMetricRegistry} for per-client and dispatcher-wide metrics. */
     private static TaggedMetricRegistry registry = DefaultTaggedMetricRegistry.getDefault();
 
-    private static final String DISPATCHER_QUEUED_CALLS_METRIC = "dispatcher.queued-calls";
-    private static final String DISPATCHER_RUNNING_CALLS_METRIC = "dispatcher.running-calls";
+    /** Shared connection pool. */
+    private static final ConnectionPool connectionPool = new ConnectionPool(100, 10, TimeUnit.MINUTES);
 
     static {
         dispatcher = new Dispatcher(executionExecutor);
         dispatcher.setMaxRequests(256);
         dispatcher.setMaxRequestsPerHost(256);
         // metrics
         registry.gauge(
-                MetricName.builder().safeName(DISPATCHER_QUEUED_CALLS_METRIC).build(),
+                MetricName.builder().safeName("dispatcher.calls.queued").build(),
```

Comment:
Would prefer to keep these scoped to `com.palantir.remoting3` or `okhttp3.Dispatcher` otherwise all libraries might end up clobbering each other!